homeassistant/components/shelly/light.py

```diff
@@ -119,6 +120,16 @@ def async_setup_rpc_entry(
     light_key_ids = get_rpc_key_ids(coordinator.device.status, "light")
     if light_key_ids:
         async_add_entities(RpcShellyLight(coordinator, id_) for id_ in light_key_ids)
+        return
+
+    rgb_key_ids = get_rpc_key_ids(coordinator.device.status, "rgb")
+    if rgb_key_ids:
```

Comment:
```suggestion
    if rgb_key_ids := get_rpc_key_ids(coordinator.device.status, "rgb"):
```
Don't forget the walrus!